pandas/tslib.pyx

```diff
@@ -3001,37 +3001,37 @@ cdef PyTypeObject* td_type = <PyTypeObject*> Timedelta
 cdef inline bint is_timedelta(object o):
     return Py_TYPE(o) == td_type # isinstance(o, Timedelta)
 
-def array_to_timedelta64(ndarray[object] values, unit='ns', errors='raise'):
-    """ convert an ndarray to an array of ints that are timedeltas
-        force conversion if errors = 'coerce',
-        else will raise if cannot convert """
+cpdef array_to_timedelta64(ndarray[object] values, unit='ns', errors='raise'):
+    """
+    Convert an ndarray to an array of timedeltas. If errors == 'coerce',
+    coerce non-convertible objects to NaT. Otherwise, raise.
+    """
+
     cdef:
         Py_ssize_t i, n
         ndarray[int64_t] iresult
-        bint is_raise=errors=='raise', is_ignore=errors=='ignore', is_coerce=errors=='coerce'
-
-    assert is_raise or is_ignore or is_coerce
```

Comment:
sure but this still could use an assert
this helps future readers know the possibilities and avoid misteps